File: app/src/main/java/com/appsys/android/popularmovie/SettingsFragment.java
package com.appsys.android.popularmovie;

import android.os.Bundle;
import androidx.preference.PreferenceFragmentCompat;

/**
 * Created by shakir on 8/7/2017.
 */

public class SettingsFragment extends PreferenceFragmentCompat {
    @Override
    public void onCreatePreferences(Bundle savedInstanceState, String rootKey) {
        addPreferencesFromResource(R.xml.pref_visualizer);
    }
}

File: app/src/main/java/com/appsys/android/popularmovie/SettingsFragment.kt
package com.appsys.android.popularmovie

import android.os.Bundle
import androidx.preference.PreferenceFragmentCompat
import com.shakirfattani.course.movielisting.R

/**
 * Created by shakir on 8/7/2017.
 */
class SettingsFragment : PreferenceFragmentCompat() {
    override fun onCreatePreferences(savedInstanceState: Bundle, rootKey: String) {
        addPreferencesFromResource(R.xml.pref_visualizer)
    }
}